src/diffusers/models/attention_processor.py

```diff
@@ -15,6 +15,7 @@
 
 import torch
 import torch.nn.functional as F
+from einops import rearrange
```

Comment:
We can't have a dependency on `einops` could we try to use it just using native PyTorch ? :-)

Reply:
Done!